### PR TITLE
Dmitrii_Matvienko Tasks 51-60

### PR DESCRIPTION
**Pull request by MatvienkoDO**

### CODE REVIEW

**Comment by MatvienkoDO** [comment]
Add examples (@example)

File: tasks/51_abbreviated-name/abbreviated-name.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/**
+ * makes abbreviated form of name
```
Comment (vovcyan) [inline tasks/51_abbreviated-name/abbreviated-name.js]: Please add example

**Review by vovcyan — CHANGES_REQUESTED** [review]

File: tasks/51_abbreviated-name/abbreviated-name.js (hunk 1 above)
Comment (MatvienkoDO) [reply tasks/51_abbreviated-name/abbreviated-name.js]: Yes, sure. I wrote for myself comment (https://github.com/MatvienkoDO/ui-internship-lwo2/pull/6#issuecomment-532212956). Working on it

**Review by alexkond — DISMISSED** [review]

File: tasks/52_sum-all-numbers-in-range/sum-all-numbers-in-range.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ * @param {Array<number>} param0
+ * @return {number} sum
+ * @example sumAll([1, 4]) = 10 (1 + 2 + 3 + 4)
+ */
+export function sumAll([firstInArr, secondInArr]) {
+  const first = Math.min(firstInArr, secondInArr);
+  const last = Math.max(firstInArr, secondInArr);
+
+  const roundedFirst = Math.ceil(first);
+  const roundedLast = Math.floor(last);
```
Comment (savchukm) [inline tasks/52_sum-all-numbers-in-range/sum-all-numbers-in-range.js]: Do we really need `ceil` and `floor` function using here?

File: tasks/57_pig-latin/pig-latin.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ */
+export function translatePigLatin(text) {
+  const first = text[0];
+
+  if (isVowel(first)) {
+    return `${text}way`;
+  }
+
+  const textLength = text.length;
+  for (let i = 0; i < textLength && !isVowel(text[0]); ++i) {
```
Comment (savchukm) [inline tasks/57_pig-latin/pig-latin.js:21]: You already have` isVowel(first)` above. What is reason of checking it in for loop? Moreover `for` is checking this condition on every condition but the result the same

**Review by savchukm — CHANGES_REQUESTED** [review]

File: tasks/52_sum-all-numbers-in-range/sum-all-numbers-in-range.js (hunk 2 above)
Comment (MatvienkoDO) [reply tasks/52_sum-all-numbers-in-range/sum-all-numbers-in-range.js]: there is no information about format of numbers. Whether it is integer or fractional

File: tasks/57_pig-latin/pig-latin.js (hunk 3 above)
Comment (MatvienkoDO) [reply tasks/57_pig-latin/pig-latin.js:21]: On every iteration it places first letter into the end of string. So, on every iteration text[0] is different

**Review by KalachevDev — DISMISSED** [review]

**Review by vovcyan — APPROVED** [review]

**Review by savchukm — APPROVED** [review]

**Pull request merged**